<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>First Run</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>5</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>This Appears to be the first time you have run OpenMolar on this computer. 
You need to enter some information before we can proceed.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_1" >
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="4" >
        <widget class="QLabel" name="mainPassword_label" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Please enter a password to prevent unauthorised running of this application.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="main_password_lineEdit" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QCheckBox" name="mainpassword_checkBox" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>show password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>82</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>115</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3" >
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6" >
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label_13" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Where is your mysql server located?</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3" >
        <widget class="QPushButton" name="ping_pushButton" >
         <property name="text" >
          <string>Test for Available Server</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="host_lineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>localhost</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QLineEdit" name="port_lineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>3306</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_14" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Port</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2" >
        <widget class="QPushButton" name="pushButton_8" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>214</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>103</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_12" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Host</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_13</zorder>
      <zorder>ping_pushButton</zorder>
      <zorder>host_lineEdit</zorder>
      <zorder>port_lineEdit</zorder>
      <zorder>label_14</zorder>
      <zorder>pushButton_8</zorder>
      <zorder>horizontalSpacer_3</zorder>
      <zorder>verticalSpacer_3</zorder>
      <zorder>label_12</zorder>
     </widget>
     <widget class="QWidget" name="page_5" >
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="1" >
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>52</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Would you like to create a demo openmolar database now?</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="createDB_pushButton" >
         <property name="text" >
          <string>Yes please</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="haveDB_pushButton" >
         <property name="text" >
          <string>No, I have a database already.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer name="verticalSpacer_4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>52</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4" >
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label_9" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Please enter a mysql user who has privileges for the openmolar database.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Database Name</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="database_lineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>openmolar_demo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>User</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="user_lineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>user</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Password</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="password_lineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>password</string>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QCheckBox" name="dbpassword_checkBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>display password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3" >
        <widget class="QPushButton" name="testDB_pushButton" >
         <property name="text" >
          <string>Test connection to this database</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>97</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="2" >
        <widget class="QPushButton" name="pushButton_9" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <layout class="QGridLayout" name="gridLayout_6" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>OK.... you are all set to go.
You will be presented with a login screen.
I trust you remember your password?

If you created a demo database... 
enter user into the user field of the login screen.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="saveQuit_pushButton" >
         <property name="text" >
          <string>Save settings and launch openmolar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" colspan="4" >
        <widget class="QLabel" name="mainPassword_label_2" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>To create a database, please enter the password of your ROOT mysql user.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="rootPassword_lineEdit" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QCheckBox" name="rootPassword_checkBox" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>show password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer name="verticalSpacer_6" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>82</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="4" >
        <widget class="QPushButton" name="createDB_pushButton_2" >
         <property name="text" >
          <string>Create openmolar_demo database, and continue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>main_password_lineEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>host_lineEdit</tabstop>
  <tabstop>port_lineEdit</tabstop>
  <tabstop>ping_pushButton</tabstop>
  <tabstop>haveDB_pushButton</tabstop>
  <tabstop>createDB_pushButton</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>database_lineEdit</tabstop>
  <tabstop>user_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>dbpassword_checkBox</tabstop>
  <tabstop>testDB_pushButton</tabstop>
  <tabstop>saveQuit_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>main_password_lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rootPassword_lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>createDB_pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>139</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
